gradle.properties

```diff
@@ -1,10 +1,10 @@
 # Library versions
 apachePoiVersion=5.2.2
-gradleAggregateJavadocPluginVersion=6.4.3
+gradleAggregateJavadocPluginVersion=6.5.1
 gradleJgitverPluginVersion=0.10.0-rc03
 gradlePublishPluginVersion=1.1.0
-modelMapperVersion=3.0.0
-springBootVersion=2.6.6
+modelMapperVersion=3.1.0
+springBootVersion=2.7.3
```

Comment:
Maybe we should upgrade to the 2.7.4 version released a few days ago. I can review it today and we can merge it.

Reply:
Upgraded, also added Apache POI since new version is now available.